pynuodb/entity.py

```diff
@@ -606,12 +606,7 @@ def start_storage_manager(self, db_name, archive, initialize, options=None, wait
         if not options:
             options = []
 
-        options.append(("--archive", archive))
-
-        if initialize:
-            options.append(("--initialize", None))
-
-        return self.__start_process(db_name, options, wait_seconds)
+        return self.__start_process(db_name, options + [("--archive", archive)] + ([("--initialize", None)] if initialize else []), wait_seconds)
```

Comment:
In 2.3, SSMs take two extra params, a `--snapshot` switch and `--snapshot-archive`. So this expression will become pretty long and unreadable when I merge it with [my changes](https://github.com/adriansuarez/nuodb-python/commit/d51434f4a5a382b9a494e399d090df114fa9a86b#diff-91374b5c1914db19d0f89b0ac113498eR581). Why don't we just make a copy of options and append to that?